packages/renderer/src/lib/preferences/PreferencesPage.svelte

```diff
@@ -37,50 +38,51 @@ onMount(async () => {
 });
 </script>
 
-<NavPage searchEnabled="{false}" title="Settings" subtitle="&nbsp;">
-  <div slot="empty" class="flex h-full px-3 py-3 bg-zinc-700">
-    <Route path="/">
-      {#if defaultPrefPageId !== undefined}
-        <PreferencesRendering key="{defaultPrefPageId}" properties="{properties}" />
-      {:else}
-        empty
-      {/if}
-    </Route>
-    <Route path="/default/:key" let:meta>
-      <PreferencesRendering key="{meta.params.key}" properties="{properties}" />
-    </Route>
-    <Route path="/ddExtensions" let:meta>
-      <PreferencesPageDockerExtensions />
-    </Route>
-    <Route path="/extensions" let:meta />
-    <Route path="/extension/:extensionId" let:meta>
-      <PreferencesExtensionRendering extensionId="{meta.params.extensionId}" />
-    </Route>
-    <Route path="/provider/:providerInternalId" let:meta>
-      <PreferencesProviderRendering providerInternalId="{meta.params.providerInternalId}" properties="{properties}" />
-    </Route>
-    <Route path="/registries">
-      <PreferencesRegistriesEditing />
-    </Route>
-    <Route path="/proxies">
-      <PreferencesProxiesRendering />
-    </Route>
-    <Route path="/extensions">
-      <ExtensionList />
-    </Route>
+<div class="flex h-full px-3 py-3 bg-zinc-900">
```

Comment:
Probably it could be a separate PR removing the header for settings (it's not specific to the resources page)
so it could be easier to check/merge ensuring the new background color is fitting well with all settings

Reply:
Opened https://github.com/containers/podman-desktop/pull/1670